docusaurus/docs/tutorials/build-an-app-plugin.md

```diff
@@ -0,0 +1,186 @@
+---
+id: build-an-app-plugin
+title: Build an app plugin
+sidebar_position: 10
+description: Learn how to create an app plugin.
+keywords:
+  - grafana
+  - plugins
+  - plugin
+  - app
+---
+
+import CreatePlugin from '@shared/create-plugin-frontend.md';
+import PluginAnatomy from '@shared/plugin-anatomy.md';
+
+## Introduction
+
+In this tutorial you will learn how to create _app plugins_. App plugins are Grafana plugins that allow you to bundle data sources and panel plugins within a single package. They also enable you to create custom pages within Grafana, providing a dedicated space for documentation, sign-up forms, or integration with other services via HTTP.
```

Comment:
could reference scenes and tabs for those pages here

Reply:
@sympatheticmoose I am not sure what you mean here. What tabs do you refer to?

Reply:
https://grafana.com/developers/scenes/scene-app-tabs